docs/section4/4-15.md

```diff
@@ -0,0 +1,53 @@
+- Eliminate duplicate HVAC systems: `HVAC_systems_virtual_list_p = list(set(HVAC_systems_virtual_list_p))`  
+Create a list of the zones served by the HVAC systems in which virtual heating and/or cooling has been modeled in the proposed RMR
+- Get dictionary with list of zones associated with each HVAC system: `dict_hvac_sys_zones_served_p = get_hvac_zone_list_w_area(P_RMR)`
+- Loop through the HVAC systems with virtual heating and/or cooling and add to list of zones with virtual heating and/or cooling: `For hvac_p in HVAC_systems_virtual_list_p:`
+    - Get list of zones associated with the hvac system and add to list of zones with virtual heating and/or cooling: `zones_virtual_heating_cooling_list = zones_virtual_heating_cooling_list.append(list(dict_hvac_sys_zones_served_p[hvac_p.id]["Zone_List"].values()))`
+- Eliminate duplicate zones from the list: `zones_virtual_heating_cooling_list = list(set(zones_virtual_heating_cooling_list))`
+- Loop through each zone in the zones_virtual_heating_cooling_list and obtain the hvac system serving the zone in the B_RMR and add to the list of HVAC systems inapplicable to this check: `For zone_p in zones_virtual_heating_cooling_list:`
+    - Get analogous zone in the B_RMR: `zone_b = match_data_element(B_RMR, Zone,zone_p.id)`  
+    - Get list of hvac systems serving the zone and add to list of inapplicable hvac systems: `inapplicable_hvac_with_virtual_heating_cooling = inapplicable_hvac_with_virtual_heating_cooling.append(get_list_hvac_systems_associated_with_zone(B_RMR, zone_b.id))`  
+- Eliminate duplicates in list of hvac systems: `inapplicable_hvac_with_virtual_heating_cooling = list(set(inapplicable_hvac_with_virtual_heating_cooling))`
+- For each HeatingVentilationAirconditioningSystem in the B_RMR check if this check is applicable: `for each hvac_b in B_RMR..HeatingVentilationAirConditioningSystem:`
+    - Check if hvac_b is not in the list of inapplicable systems, if it is not found then set applicability flag to true: `if hvac_b not in inapplicable_hvac_with_virtual_heating_cooling: rule_applicability_flag = TRUE`   
+    - If rule_applicability_flag = TRUE for hvac_b: `if rule_applicability_flag == TRUE:`
+        - Get fan operation during occupied periods: `operation_during_occupied_b = hvac_b.fan_systems[0].operation_during_occupied` 
+        - Get minimum_outdoor_airflow: `minimum_outdoor_airflow_b = hvac_b.fan_systems[0].minimum_outdoor_airflow`
+
+        **Rule Assertion:**
+        - Case 1: For each applicable hvac system, if fan system operation during occupied periods is continuous and the minimum OA is greater than 0 in the B_RMR: `if operation_during_occupied_b == “CONTINUOUS” and minimum_outdoor_airflow_p >0 : PASS`
+        - Case 2: else if, fan system operation during occupied periods is cycling and the minimum OA equals 0 in the B_RMR: `else if operation_during_occupied_b == “CYCLING” and minimum_outdoor_airflow_p == 0 : PASS`
+        - Case 3: Else: `Else: FAIL`
```

Comment:
@mkarpman not sure why the cycling check is required. Cycling fan with OA =0 would only happen if a system is coupled with DOAS, correct? And that wouldn't happen with the baseline. Do we need case 2 at all?